abb_experimental.rosinstall

```diff
@@ -0,0 +1,5 @@
+#Temporary until PR accepted by https://github.com/ros-industrial/
+- git: {local-name: abb, uri: 'https://github.com/AustinDeric/abb.git', version: kinetic-devel}
```

Comment:
Only until we get the ros-industrial/abb kinetic-devel branch up and running.  My intent was for this to be merged after that so we can make the change in this file.  In hid-sight i should have said this in my description, i will fix.  

The reason i did it like this is so that we can test it in travis.